projects/radare2/default.options

```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+close_fd_mask=3
+detect_leaks=0
```

Comment:
This deserves a comment. Are there real leaks? 
If the fuzzer finds leaks often, the fuzzing won't be efficient due to induced OOMs.

Reply:
Yes there are real leaks. I'm now in contact with the project owner to see if those leaks can get fixed. I will update the PR when it's done.